dune_daq_services/runservices/runnumber-rest/runnumber-rest.yaml

```diff
@@ -26,19 +25,20 @@ spec:
            - matchExpressions:
              - key: {{ kubernetes_label.worker }}
                operator: Exists
+      initContainers:
```

Comment:
I'm not clear this is necessary as it should respawn automatically if postgresql isn't up yet.

Reply:
It does respawn automatically, but in the old version of pocket we had something similar so that there wasn't a need for a restart, and I assumed since Pierre asked me to do this last time, he'd have wanted it again in this one. But if you feel like its too much, I'm happy to take it out